dm/dm/config/task_cli_args.go

```diff
@@ -28,7 +28,9 @@ const (
 // TaskCliArgs is the task command line arguments, these arguments have higher priority than the config file and
 // downstream checkpoint, but may need to be removed after the first time they take effect.
 type TaskCliArgs struct {
-	StartTime string `toml:"start-time" json:"start_time"`
+	StartTime              string `toml:"start-time" json:"start_time"`
+	SafeModeDuration       string `toml:"safe-mode-duration" json:"safe_mode_duration"`
+	WaitTimeOnStopDuration string `toml:"wait-time-on-stop-duration" json:"wait_time_on_stop_duration"`
```

Comment:
`duration` looks duplicate with `wait-time`

Reply:
added in https://github.com/pingcap/tiflow/pull/4601/commits/5334eaf152d9ca3018135d5a4c7144f55ae93773